skytemple_randomizer/frontend/gtk/main.py

```diff
@@ -145,6 +145,10 @@ def main(argv: list[str] | None = None):
             Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION,
         )
 
+    if sys.platform == "darwin":
```

Comment:
lmk if there's a better place for this code

Reply:
Can you use `platform.system() == "Darwin"`? Trying to make this a bit more consistent and that's kinda the API to use for this.